vm/src/utils.rs

```diff
@@ -76,7 +76,7 @@ pub mod test_utils {
             Into::<num_bigint::BigInt>::into($val)
         };
     }
-    pub(crate) use bigint;
+    pub use bigint;
```

Comment:
This is not really vm-related, so I don't think it should be included in the vm's public api. Moreover, we may choose to remove the usage of BigInt from the VM altogether and remove this macro too.

Reply:
Makes sense. I made it public out of simplicity because keeping it `pub(crate)` in a now public module induces a few unused import warnings. I'll find a solution.